tools/ci/run_tc.py

```diff
@@ -257,13 +257,33 @@ def setup_environment(args):
         assert args.channel is not None
         install_chrome(args.channel)
 
+    if "chromium" in args.browser:
+        install_chromium()
+
     if args.xvfb:
         start_xvfb()
 
     if args.oom_killer:
         start_userspace_oom_killer()
 
 
+def install_chromium():
+    revisions_path = os.path.join(os.getcwd(), "tools", "wpt", "latest_chromium_revision.txt")
+    with open(revisions_path) as f:
+        revision = f.read().strip()
+        dest = os.path.join("/tmp")
+        installer_path = os.path.join("/tmp", "chrome-linux.zip")
+
+        url = ("https://storage.googleapis.com/chromium-browser-snapshots/Linux_x64/"
+               f"{revision}/chrome-linux.zip")
+        resp = get(url)
+        with open(installer_path, "wb") as f:
+            f.write(resp.content)
```

Comment:
Does this end up reading the whole file into memory and then to disk? Can it be streamed to disk, or unzipped directly from the network? Given how big the archives are it seems like it could make a measurable difference. Worth timing this before doing too much work optimizing it.

Reply:
I'll take a look into it. Additionally, I think it might be worth it to remove this CI installation change entirely from this PR so that we can implement everything in a separate PR involved in changing to using Chromium for the CI.